integration-tests/spring-di/src/main/java/io/quarkus/it/spring/MyConfigurationWithNamedBean.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.it.spring;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MyConfigurationWithNamedBean {
+
+    @Bean(name = "mySpecialName")
```

Comment:
Could be applied via default param like done with `@Qualifier("mySpecialName")`
```suggestion
    @Bean("mySpecialName")
```